src/wp-includes/block-template-utils.php

```diff
@@ -648,14 +655,15 @@ function _wp_build_title_and_description_for_taxonomy_block_template( $taxonomy,
 		$term_title
 	);
 
-	$terms_with_same_title = get_terms(
-		array(
-			'taxonomy'   => $taxonomy,
-			'hide_empty' => false,
-			'name'       => $term_title,
-		)
+	$args = array(
+		'taxonomy'               => $taxonomy,
+		'hide_empty'             => false,
+		'name'                   => $term_title,
+		'update_post_term_cache' => false,
```

Comment:
Should this have `number => 2` as the test is for the existence of duplicates but doesn't indicate how many there are? Is it worth it?